### ISSUE TITLE
TypeError: Cannot read property 'fsPath' of undefined

### ISSUE THREAD

**Issue by hrvoj3e** [zabel-xyz/local-history#35]
User setting
```
    "files.exclude": {
        "**/.history/**": true,
        ".vscode/.history/**": true
    },
    "local-history.path": "${workspaceFolder}/.vscode",
```

Console log
```
Cannot read property 'fsPath' of undefined: TypeError: Cannot read property 'fsPath' of undefined
	at HistorySettings.read (C:\Users\Hrvoje\.vscode\extensions\xyz.local-history-1.6.2\out\src\history.settings.js:93:74)
	at HistorySettings.get (C:\Users\Hrvoje\.vscode\extensions\xyz.local-history-1.6.2\out\src\history.settings.js:36:29)
	at HistoryController.getSettings (C:\Users\Hrvoje\.vscode\extensions\xyz.local-history-1.6.2\out\src\history.controller.js:68:30)
	at resolve (C:\Users\Hrvoje\.vscode\extensions\xyz.local-history-1.6.2\out\src\historyTree.provider.js:43:54)
	at Promise (<anonymous>)
	at HistoryTreeProvider.getChildren (C:\Users\Hrvoje\.vscode\extensions\xyz.local-history-1.6.2\out\src\historyTree.provider.js:23:16)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:457:859
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:98:622
	at new n.Class.derive._oncancel (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:75:657)
	at Object.t.asWinJsPromise (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:98:585)
e.doShow @ /C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:2925
/C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4185 [xyz.local-history]Cannot read property 'fsPath' of undefined
e.$onExtensionRuntimeError @ /C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4185
```

VS Code
```
Version 1.20.0
Commit c63189deaa8e620f650cc28792b8f5f3363f2c5b
Date 2018-02-07T17:09:39.780Z
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
```

![2018-02-13 113537 screenshot](https://user-images.githubusercontent.com/4988133/36145559-0671e046-10b2-11e8-956c-905b1b842907.png)

**Comment by zabel-xyz** [comment]
This exception appears without a workspace but with config like `"local-history.path": "${workspaceFolder}/.vscode"`

I'll fix it...

**Comment by zabel-xyz** [comment]
Fix in 1.7.0
